.github/workflows/release.yml

```diff
@@ -32,11 +32,10 @@ jobs:
       -
         name: Import GPG key
         id: import_gpg
-        uses: dmitrykruglov/ghaction-import-gpg@main
+        uses: crazy-max/ghaction-import-gpg@v5
         env:
```

Comment:
👍 looks good- just a small note that `env:` should be updated to `with:`

Reply:
thank you